kunstakademie-munster.csl

```diff
@@ -188,72 +213,135 @@
             <label form="short" prefix=" "/>
           </names>
-          <text variable="container-title" font-style="italic"/>
-          <!-- Italic für Kapitel in Sammelbänden -->
-          <group delimiter="; " prefix=" (" suffix=")">
-            <text variable="collection-title"/>
-            <text variable="collection-number" prefix="Band "/>
-          </group>
-          <text macro="volume" prefix="; "/>
-          <text variable="publisher-place" prefix=", "/>
-          <!--         <text variable="publisher" prefix=": "/> -->
+          <text variable="container-title" font-style="italic" suffix=","/>
+          <text variable="publisher-place" prefix=" "/>
           <text value=" "/>
           <text variable="edition" vertical-align="sup"/>
           <text macro="year-date"/>
-          <text variable="page" prefix=", S. "/>
+          <text macro="volume"/>
+	        <text variable="page" prefix=", S. "/>
         </else-if>
+
         <else>
-          <text macro="editor-transl-long" prefix="; "/>
-          <text macro="volume" prefix="; "/>
-          <group delimiter="; " prefix=" (" suffix=")">
-            <text variable="collection-title"/>
-            <text variable="collection-number" prefix="Band "/>
-          </group>
+          <text macro="editor-transl-long" prefix="; "/>      
           <text variable="publisher-place" prefix=", "/>
-          <!--          <text variable="publisher" prefix=": "/>	 -->
-          <!-- Verlag raus -->
           <text value=" "/>
           <text variable="edition" vertical-align="sup"/>
           <text macro="year-date"/>
-          <text variable="note" prefix=" "/>
-          <text variable="number-of-pages" prefix=", " suffix=" S."/>
-          <text variable="ISBN" prefix=", ISBN "/>
-        </else>
+          <text macro="volume"/>
+         </else>		
+
       </choose>
+</macro>
+
+<macro name="pagenumber">
+  <choose>
+    <if locator="page" match="any">
+      <choose>
+        <if position="first">
+          <choose>
+            <if variable="page" match="any">
+              <text term="first-position" prefix=", " suffix=" "/>
+              <label variable="locator"  form="short" prefix=" " suffix=" "/>
+            </if>
+             <else>
+              <label variable="locator"  form="short" prefix=", " suffix=" "/>
+            </else>
+          </choose>
+        </if>
+        <else>
+          <label variable="locator"  form="short" prefix=", " suffix=" "/>
+        </else>
+      </choose> 
+      <text variable="locator"/>
+    </if>
+    <else>
+      <group prefix=", ">
+        <text variable="locator"/>
+      </group>
+    </else>
+  </choose>
+</macro>
+
+  <!-- ====== ZITATE =============================== -->
+ <citation et-al-min="3" et-al-use-first="2">
+    <layout suffix="." delimiter="; " disambiguate-add-year-suffix="true">
```

Comment:
disambiguate-add-year-suffix="true" not allowed on layout. put it on citation/bibliography.